packages/xdg_base.rb

```diff
@@ -3,41 +3,45 @@
 class Xdg_base < Package
   description 'XDG Base Directory Specification Configuration'
   homepage 'https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html'
-  version '0.7-2'
-  source_url 'https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html'
-  source_sha256 '342289ad33b48b8f643278f74cafa182f4471d490b82dc5e442e2e720fa4080f'
+  version '0.7-3'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
   def self.preinstall
     # Save any previous configuration, if it exists.
-    if Dir.exists? "$HOME/.config"
-      system "cp -a $HOME/.config #{CREW_PREFIX} && rm -rf $HOME/.config"
+    if Dir.exists? "#{HOME}/.config"
+      FileUtils.cp_r("#{HOME}/.config", "#{CREW_PREFIX}/")
+      FileUtils.rm_rf("#{HOME}/.config")
     end
-    if Dir.exists? "$HOME/.local"
-      system "mkdir -p #{CREW_PREFIX}/.config" unless Dir.exists? "#{CREW_PREFIX}/.config"
-      system "cp -a $HOME/.local/. #{CREW_PREFIX}/.config && rm -rf $HOME/.local"
```

Comment:
There is a reason the `&&` is here.  If for any reason the `cp -a ...` fails, the `rm -rf ...` statement is ignored.  Otherwise, this could lead to potential lost data.

Reply:
OK.

Reply:
Would it work to use what I'm doing, but concatenate the lines with an `if` in between?